src/lib/stitching/vortex/stitching.ts

```diff
@@ -44,9 +45,6 @@ export const vortexStitchingEnvironment = (localSchema: GraphQLSchema) => ({
     extend type Partner {
       analytics: AnalyticsPartnerStats
     }
-    extend type AnalyticsTopArtworks {
-      artwork: Artwork
-    }
```

Comment:
`TopArtworks` is now only deprecated in Vortex. Would removing the stitching here make clients querying the stitched `artwork` fail?

Reply:
Yes it would.

The deprecated `topArtwork` in vortex now returns `null` so leaving this here could also create error in MP.

Since this feature is not on production based on our previous discussion around 'fail fast and fix' we can quickly follow up with Volt PR and fix this.

Reply:
Commented https://github.com/artsy/vortex/pull/149/files#r293502277, and I wonder if we should keep it for backward compatibility and exercise removal of deprecated stuff separately later.

Reply:
Also cross posting: https://github.com/artsy/vortex/pull/149/files#r293509762